amethyst_renderer/src/pipe/effect/pso.rs

```diff
@@ -102,9 +112,10 @@ impl<'d> PipelineInit for Init<'d> {
     }
 }
 
-#[derive(Clone, Debug, Default, Eq, Hash, PartialEq)]
+#[derive(Clone, Debug, Default, PartialEq)]
 pub struct Data {
     pub const_bufs: Vec<<RawConstantBuffer as DataBind<Resources>>::Data>,
```

Comment:
hmm, not sure it has to get all the `Data` from the component types. Couldn't you just specify the types directly, and what would be the downside of this?